### ISSUE TITLE
Broken colab notebook

### ISSUE THREAD

**Issue by r9y9** [r9y9/nnsvs#78]
Not a priority, but I will fix it at some point.

**Comment by nicolalandro** [comment]
I think that another fix is to correctly install pysinsy by update submodule before install

```
! cd pysinsy && git submodule update --recursive --init && export SINSY_INSTALL_PREFIX=/usr/ && pip install -q .
```

**Comment by nicolalandro** [comment]
Now I have the problem at Time-lag model

```
---------------------------------------------------------------------------

ConfigAttributeError                      Traceback (most recent call last)

[<ipython-input-16-1af2a256b7d5>](https://localhost:8080/#) in <module>()
      1 timelag_config = OmegaConf.load(join(model_dir, "timelag", "model.yaml"))
----> 2 timelag_model = hydra.utils.instantiate(timelag_config.netG).to(device)
      3 checkpoint = torch.load(join(model_dir, "timelag", "latest.pth"), map_location=lambda storage, loc: storage)
      4 timelag_model.load_state_dict(checkpoint["state_dict"])
      5 timelag_in_scaler = joblib.load(join(model_dir, "in_timelag_scaler.joblib"))

6 frames

[/usr/local/lib/python3.7/dist-packages/omegaconf/dictconfig.py](https://localhost:8080/#) in _get_node(self, key, validate_access, throw_on_missing_value, throw_on_missing_key)
    468         if value is None:
    469             if throw_on_missing_key:
--> 470                 raise ConfigKeyError(f"Missing key {key}")
    471         elif throw_on_missing_value and value._is_missing():
    472             raise MissingMandatoryValue("Missing mandatory value: $KEY")

ConfigAttributeError: Missing key to
    full_key: netG.to
    object_type=dict
```

**Comment by nicolalandro** [comment]
Also hts_engine do not install correctly, I try to do the following commands:

```
! cd hts_engine_API/src && mkdir -p build && cd build && cmake .. && cmake --build . --config Release
```

And also sinsy fail the install

**Comment by nicolalandro** [comment]
In that modified notebook seams that the installation of the libraries works correctly [Neural_network_based_singing_voice_synthesis_demo_using_kiritan_singing_database_(Japanese).zip](https://github.com/r9y9/nnsvs/files/8230609/Neural_network_based_singing_voice_synthesis_demo_using_kiritan_singing_database_.Japanese.zip) .

Sometimes the import box should be run two times (that can be a first smell).

After I arrive to the Time-lag model box and I reach:
```
---------------------------------------------------------------------------

ConfigAttributeError                      Traceback (most recent call last)

[<ipython-input-16-1af2a256b7d5>](https://localhost:8080/#) in <module>()
      1 timelag_config = OmegaConf.load(join(model_dir, "timelag", "model.yaml"))
----> 2 timelag_model = hydra.utils.instantiate(timelag_config.netG).to(device)
      3 checkpoint = torch.load(join(model_dir, "timelag", "latest.pth"), map_location=lambda storage, loc: storage)
      4 timelag_model.load_state_dict(checkpoint["state_dict"])
      5 timelag_in_scaler = joblib.load(join(model_dir, "in_timelag_scaler.joblib"))

6 frames

[/usr/local/lib/python3.7/dist-packages/omegaconf/dictconfig.py](https://localhost:8080/#) in _get_node(self, key, validate_access, throw_on_missing_value, throw_on_missing_key)
    468         if value is None:
    469             if throw_on_missing_key:
--> 470                 raise ConfigKeyError(f"Missing key {key}")
    471         elif throw_on_missing_value and value._is_missing():
    472             raise MissingMandatoryValue("Missing mandatory value: $KEY")

ConfigAttributeError: Missing key to
    full_key: netG.to
    object_type=dict
```

Any idea about that problem? Seams that it cannot be loaded into GPU, without the .to(device), it seams to work and break in the next instruction.

**Comment by r9y9** [comment]
Sorry for the inconvenience. I am currently working on a huge refactoring. After finishing it, I will update the notebook.

**Comment by nicolalandro** [comment]
Thank you! Can I help in some way?

**Comment by r9y9** [comment]
Sure. I'd appreciate it if you can help to test once the huge refactoring settled down.

**Comment by nicolalandro** [comment]
Ok, I will do it!

**Comment by r9y9** [comment]
Hi @nicolalandro, I've merged the #81 and made a new demo notebook. I think the installation process now became much more simpler.

- Colab link: https://colab.research.google.com/github/r9y9/nnsvs/blob/master/notebooks/Demos.ipynb
- Github link: https://github.com/r9y9/nnsvs/tree/master/notebooks

Could you check if it works for you? Also, I'd appreciate it if you make a streamlet app version of the demo! Any feedbacks or PRs are welcomed.

Side note: The notebook above focuses on Japanese singing voice synthesis. It shouldn't be difficult to make a demo for other languages. However, I have no experience using nnsvs for non-Japanese languages and not sure what's the best idea.

**Comment by nicolalandro** [comment]
Thank You! It works correctly on Colab! I close this issue and I start to work on Streamlit app like in the issue!
